Updated Example configuration for Netdata

noodlemctwoodle opened the pull request:
Updated example in documentation as per this [Github](https://github.com/home-assistant/home-assistant/pull/16656#issuecomment-422563838) post

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

noodlemctwoodle committed:
Updated Example configuration

Updated example in documentation as per this [Github](https://github.com/home-assistant/home-assistant/pull/16656#issuecomment-422563838) post

klaasnicolaas commented:
Please rebase this to the `current` branch

klaasnicolaas commented on source/_components/netdata.markdown:
```diff
@@ -49,15 +49,20 @@ To add this platform to your installation, add the following to your `configurat
-      cpu:
-        data_group: system.cpu
-        element: system
+        icon: mdi:chip
```
Only required items in the first example

noodlemctwoodle replied:
removed non required data from the example

klaasnicolaas commented on source/_components/netdata.markdown:
```diff
@@ -49,15 +49,20 @@ To add this platform to your installation, add the following to your `configurat
+      core_temp:
+        data_group: sensors.pch_skylake-virtual-0_temperature
+        element: pch_skylake-virtual-0_temp1
+        icon: mdi:thermometer-lines
```
same

noodlemctwoodle replied:
reverted example

klaasnicolaas commented on source/_components/netdata.markdown:
```diff
@@ -49,15 +49,20 @@ To add this platform to your installation, add the following to your `configurat
+      mem_avail:
+        data_group: mem.available
+        element: MemAvailable
+        icon: mdi:chip
```
same

noodlemctwoodle replied:
reverted example

fabaff commented on source/_components/netdata.markdown:
```diff
@@ -49,15 +49,20 @@ To add this platform to your installation, add the following to your `configurat
   - platform: netdata
-    resources:
-      load:
+    resources:    
```
Revert this please.

noodlemctwoodle replied:
reverted example.

fabaff reviewed (changes requested):
Additional details: https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

noodlemctwoodle committed:
Updated netdata documents

 - Added another way to read the API data in the web browser.
 - Added example and description
 - Reverted `configuration` example back to required

noodlemctwoodle committed:
corrected typo

noodlemctwoodle commented on source/_components/netdata.markdown:
````diff
@@ -43,23 +43,53 @@ $ curl -X GET "http://[Netdata_Instance]:19999/api/v1/data?chart=[data_group]&po
+```
+
+Once the `data_group` and the `element` have been identified from the JSON  it can be configured as per the example below. 
+
````
Added another example of how to collect data from the API for those unable to collect it using curl.

noodlemctwoodle reviewed (commented):
Added addition example.

noodlemctwoodle commented:
updated component

klaasnicolaas commented:
Can you rebase this PR to the `current` branch? Your changes are not related to something that will come in a next release.

jjanderson commented:
HI, is this stalled? I am trying to configure but really getting lost... an example would be great... especially if it were say based on Ubuntu 18.04? :-)

noodlemctwoodle commented:
I can assist with instructions for 18.04. I'll get the rebase done this evening.

noodlemctwoodle commented:
created new pull request here as i broke my repo :|

Updated Netdata documentation #9606
